src/webgpu/web_platform/external_texture/video.spec.ts

```diff
@@ -247,12 +248,14 @@ for several combinations of video format and color space.
 g.test('importExternalTexture,expired')
   .desc(
     `
-Tests that GPUExternalTexture.expired is false when HTMLVideoElement is not updated
-or VideoFrame(webcodec) is alive. And it will be changed to true when imported
-HTMLVideoElement is updated or imported VideoFrame is closed. Using expired
-GPUExternalTexture results in an error.
+Tests that GPUExternalTexture is valid after import until next task comes or VideoFrame(webcodec) is closed.
+In the same task scope, import twice on the same video source should return the same object.
+In the same task scope, bindGroup doesn't need to be updated.
+If next task comes, previous GPUExternalTexture should be expired. An new imported GPUExternalTexture should
+work.
 
 TODO: Make this test work without requestVideoFrameCallback support (in waitForNextFrame).
+TODO: Adding a test in requestAnimationFrame.
 `
   )
   .params(u =>
```

Comment:
This test is starting to do a lot of different things for the various ways to import an external texture. We could split it per type, with helpers to help factor in code. Then have separate tests for expiry + reimport, closing, etc.

Reply:
Updated a new form of cts, PTAL again, thanks!